src/config/config.rs

```diff
@@ -63,6 +63,11 @@ pub struct Config {
     #[serde(default, skip_serializing_if = "is_default")]
     pub unions: BTreeMap<String, Union>,
 
+    ///
+    /// A map of all the enum types in the schema
+    #[serde(default, skip_serializing_if = "is_default")]
+    pub enums: BTreeMap<String, BTreeSet<String>>,
```

Comment:
Looks like we loose out on important details such as `docs` etc. Can we make this change 

```rust
pub struct Enum {
    pub variants: BTreeSet<String>,
    pub doc: Option<String>,
}
```